### ISSUE TITLE
Implement other classifiers

### ISSUE THREAD

**Issue by AidanGG** [AidanGG/TensorFlow-to-TMVA-Adapter#5]
The following classifiers still need work done on them. Given the work done with ANN, k-NN and SVM, it shouldn't be too hard to apply the same principles of mapping the problem to n-dimensional arrays, defining an unconstrained cost function and running gradient descent or some other optimiser on the cost.

Rectangular Cut Optimisation
Projective Likelihood
Multidimensional Probability Density Estimator Range-Search
Fisher and Mahalanobis Discriminants
H-Matrix Estimator
Function Discriminant Analysis
Boosted Decision Trees
Predictive Learning via Rule Ensembles

Some of these may not lend themselves well to implementation in TensorFlow (such as Boosted Decision Trees).
